vms/registration/tests/test_functional_admin.py

```diff
@@ -271,56 +308,99 @@ def test_city_field(self):
         self.driver.find_element_by_id('id_unlisted_organization').send_keys('admin-org')
         self.driver.find_element_by_xpath('//form[1]').submit()
 
+        # verify successful registration
         self.assertNotEqual(self.driver.find_elements_by_class_name('messages'),
                 None)
         self.assertEqual(self.driver.find_element_by_class_name('messages').text,
                 'You have successfully registered!')
 
-        self.assertEqual(self.driver.current_url, self.live_server_url +
```

Comment:
@tapasweni-pathak I checked again, I haven't removed it here. I removed an entire case for test_city and this line was a part of it, so it was removed. But location fields has these url checks everywhere.

I forgot to add this url check for email and phone field, I will do that with later commits, which I will push here shortly.